components/match2/wikis/deadlock/match_summary.lua

```diff
@@ -9,16 +9,19 @@
 local Abbreviation = require('Module:Abbreviation')
 local Array = require('Module:Array')
 local CharacterIcon = require('Module:CharacterIcon')
+local Class = require('Module:Class')
 local FnUtil = require('Module:FnUtil')
 local DateExt = require('Module:Date/Ext')
 local Icon = require('Module:Icon')
 local Logic = require('Module:Logic')
 local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
 local Table = require('Module:Table')
 
 local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper')
 local MatchSummary = Lua.import('Module:MatchSummary/Base')
 
+local MAX_NUM_BANS = 6
```

Comment:
imo we should write the code to do it without this fixed number

Reply:
let's leave it out of scope for now